public/locales/en/translation.json

```diff
@@ -86,6 +86,10 @@
     "userGreeting": "Hello",
     "userinfoUpdate": "Update Info"
   },
+  "userDetailsForm": {
+    "submitButton": "Update",
+    "title": "User Details"
```

Comment:
maybe better to show the meaning in the key?
"Update": "Update",    
"User Details": "User Details"